libraries/AP_RangeFinder/AP_RangeFinder.cpp

```diff
@@ -413,15 +413,20 @@ void RangeFinder::detect_instance(uint8_t instance, uint8_t& serial_instance)
                 }
             }
         break;
-    case Type::BenewakeTFminiPlus:
+    case Type::BenewakeTFminiPlus: {
+        uint8_t addr = TFMINI_ADDR_DEFAULT;
```

Comment:
```suggestion
        const uint8_t addr = TFMINI_ADDR_DEFAULT;
```

Reply:
Fixed this. Thanks @hendjoshsr71 for reviewing.

Reply:
We are updating `addr` on line 419 so that won't go well if I declare `addr` const.

Reply:
Doh.... fail on my part. Thanks!

Reply:
I can do this in single line using `?` operator while passing address to `get_device` method (`get_device(i, params[instance].address == 0 ? TFMINI_ADDR_DEFAULT : params[instance].address)`) if you say. I don't think that would look pretty tough.

Reply:
No, what you have is fine and matches the pattern for this in this function.

Reply:
Thanks for reviewing. 🙂